### PR TITLE
update code examples for spanish translation

### PR DESCRIPTION
**Pull request by dulcehc**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/spanish/03-front-end-libraries/sass/nest-css-with-sass.spanish.md [hunk 1]
```diff
@@ -10,7 +10,34 @@ localeTitle: Nest CSS con Sass
...
+  nav {
+    background-color: red;
+    ul {
+      list-style: none;
+      li {
+        display: inline-block;
+      }
+    }
+  }
+```
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/spanish/03-front-end-libraries/sass/nest-css-with-sass.spanish.md]: I believe we need `</section>` here. Will have to check once local setups are working again.

Comment (dulcehc) [reply curriculum/challenges/spanish/03-front-end-libraries/sass/nest-css-with-sass.spanish.md]: Good point, I agree.

Comment (RandellDawson) [reply curriculum/challenges/spanish/03-front-end-libraries/sass/nest-css-with-sass.spanish.md]: I corrected this for the user.

**Comment by RandellDawson** [comment]
Content LGTM.  Still needs to be tested locally.

**Comment by scissorsneedfoodtoo** [comment]
@dulcehc, sorry for the delay. Had to change the spacing between the sections a bit to get them to render properly. Now everything LGTM! Thank you for your contribution :+1: :+1:

**Pull request merged**